packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -74,672 +94,2415 @@ describe("@fulltext directive", () => {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst].name).toBe(person1.name);
+            expect((gqlResult.data?.[queryType] as any[])[1][personTypeLowerFirst].name).toBe(person3.name);
+            expect((gqlResult.data?.[queryType] as any[])[0][SCORE_FIELD]).toBeGreaterThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]
+            );
+            expect(gqlResult.data?.[queryType] as any[]).toBeArrayOfSize(2);
+        });
+        test("Filters score to no results", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "a different name", where: { score: { min: 100 } }) {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect(gqlResult.data?.[queryType]).toEqual([]);
+        });
+        test("Filters score with combined min and max", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "a different name", where: { score: { min: 0.201, max: 0.57 } }) {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst].name).toBe(person1.name);
+            expect((gqlResult.data?.[queryType] as any[])[0][SCORE_FIELD]).toBeNumber();
+            expect(gqlResult.data?.[queryType] as any[]).toBeArrayOfSize(1);
+        });
+        test("Filters score with max score of 0", async () => {
```

Comment:
Just an opinionated stylistic comment for future reference, it would be nice to see a bit of whitespace between tests to make them just a bit easier to read!